### ISSUE TITLE
Discussion: Change android support dependency to allow jetifier removal

### ISSUE THREAD

**Issue by tpucci** [microsoft/appcenter-sdk-android#1505]
Hi!

We are trying to get rid of jetifier on our Android project.
However, it seems appcenter sdk 4.1.0 still uses legacy android support libraries.

## bye-bye-jetifier logs
```
=========================================
Project: app
=========================================

Scanning com.microsoft.appcenter:appcenter:4.1.0
 Absoulute path: /......./appcenter-4.1.0.aar
 Graphs to this dependency:
 +---com.microsoft.appcenter:appcenter-distribute:4.1.0
     +---com.microsoft.appcenter:appcenter:4.1.0
 Issues found:
 * com/microsoft/appcenter/utils/InstrumentationRegistryHelper.class -> android.support.test.InstrumentationRegistry

> Task :canISayByeByeJetifier FAILED
```

Is it possible to replace this library?
I guess the issue comes from the following line:
https://github.com/microsoft/appcenter-sdk-android/blob/de5082da8b9955139596efbf7c444acec4755117/sdk/build.gradle#L86

**Comment by Jamminroot** [comment]
Hey, @tpucci, thanks for bringing this to our sight. Please let us some time to have a look at this on our side and discuss it with the team.

**Comment by AnastasiaKubova** [comment]
The fix has been released in the [4.2.0](https://github.com/microsoft/appcenter-sdk-android/releases/tag/4.2.0) version of App Center SDK. Please update the App Center SDK to the latest version and let me know if this issue still persists.

**Comment by tpucci** [comment]
Thanks !
